nexus/src/app/silo.rs

```diff
@@ -27,12 +29,11 @@ impl super::Nexus {
     // Silos
 
     pub async fn silo_create(
-        &self,
+        self: &Arc<Self>,
```

Comment:
I assume this is a holdover from when this created a saga.

Reply:
Yeah, removed in 35962ce77a41088497ff20d5a7f4553744202326